control-plane-operator/hostedclusterconfigoperator/controllers/resources/resources.go

```diff
@@ -1509,6 +1511,9 @@ func (r *reconciler) ensureCloudResourcesDestroyed(ctx context.Context, hcp *hyp
 	}
 	if !removed {
 		remaining.Insert("ingress-controllers")
+		return remaining, errors.NewAggregate(errs)
+	} else {
+		log.Info("Ingress controllers are removed")
 	}
```

Comment:
another approach would be to exclude the ingress LB from `ensureServiceLoadBalancersRemoved` and let the ingress controller deletion to take care of it, so then we don't need ordering.

Reply:
Thanks, that makes sense. Updated to do that.